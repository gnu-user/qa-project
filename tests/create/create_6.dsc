TITLE
    Creating a User (Enter User Type Prompt)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the create 
    transaction. Enter a valid username, and verify that the user is prompted 
    to enter a user type.
DEPENDENCIES
    login Test Suite
    create_2
