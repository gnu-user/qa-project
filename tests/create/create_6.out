Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the username to create: 
Enter the user type: 